ReactAndroid/src/main/java/com/facebook/react/views/viewpager/ReactViewPagerManager.java

```diff
@@ -41,6 +41,11 @@ public void setSelectedPage(ReactViewPager view, int page) {
     view.setCurrentItemFromJs(page);
```

Comment:
Is the comment above this line still valid? The task only has a link to this file :) It was added by Krzysztof.